containers/kubernetes/how-to/upgrade-kubernetes-version.mdx

```diff
@@ -16,7 +16,7 @@ categories:
 <Macro id="iam-requirements" />
 
 
-You can either upgrade your Kubernetes Kapsule cluster [directly from the Scaleway console](#upgrading-a-kapsule-cluster-from-the-scaleway-console), or upgrade your cluster to the next minor version by [using the CLI](#upgrading-a-kapsule-cluster-to-the-next-minor-version-using-the-cli). The CLI section of this how-to guide also provides very helpful tips for checking important components before proceeding with the upgrade from your cluster.
+You can either upgrade your Kubernetes Kapsule cluster [directly from the Scaleway console](#upgrading-a-kapsule-cluster-from-the-scaleway-console), or upgrade your cluster to the next minor version by [using the CLI](#upgrading-a-kapsule-cluster-to-the-next-minor-version-using-the-cli). The CLI section of this how-to guide also provides tips for checking important components before proceeding with the upgrade of your cluster.
```

Comment:
these are not 'tips' but mandatory constraints

Reply:
```suggestion
You can either upgrade your Kubernetes Kapsule cluster [directly from the Scaleway console](#upgrading-a-kapsule-cluster-from-the-scaleway-console), or upgrade your cluster to the next minor version by [using the CLI](#upgrading-a-kapsule-cluster-to-the-next-minor-version-using-the-cli). The CLI section of this how-to guide also provides additional mandatory information for checking important components before proceeding with the upgrade of your cluster.
```

Reply:
my bad I wrote "tips" earlier for a lack of finding a better word.